src/lib/db/seed.ts

```diff
@@ -143,6 +144,15 @@ export const seed = async (db: Database, much = false): Promise<void> => {
 		await createDefaultEntities(toMainAccountServiceRequest, spaces, nextActor, repos);
 	}
 
+	const adminHub = await repos.hub.loadAdminHub();
+	await UpdateHubService.perform({
+		...toMainAccountServiceRequest(),
+		params: {
+			actor: mainActorCreator.actor_id,
+			hub_id: ADMIN_HUB_ID,
+			settings: {...adminHub!.settings, instance: {minimumPasswordLength: 3}},
```

Comment:
The passwords we seed are 1 character long, is there a reason to make this anything else?

```suggestion
			settings: {...adminHub!.settings, instance: {minimumPasswordLength: 1}},
```